README.md

```diff
@@ -40,6 +40,89 @@ reporting to a file, supply the `-e` option a location for the file to be
 saved. By running the jar without any arguments or by providing the optional
 parameter `-h` you can view the full usage statement. 
 
+### Annotation fields
+| Field | Source | Note |
+| --| -- | -- |
+| Hugo_Symbol |||
+|Entrez_Gene_Id|||
+|Center|||
+|NCBI_Build|||
+|Chromosome|GN response / Input||
+|Start_Position|GN response / Input||
+|End_Position|GN response / Input||
+|Strand|||
+|Consequence|||
+|Variant_Classification|||
+|Variant_Type|||
+|Reference_Allele|GN response / Input||
+|Tumor_Seq_Allele1|||
+|Tumor_Seq_Allele2|GN response / Input||
+|dbSNP_RS|||
+|dbSNP_Val_Status|||
+|Tumor_Sample_Barcode|||
+|Matched_Norm_Sample_Barcode|||
+|Match_Norm_Seq_Allele1|||
+|Match_Norm_Seq_Allele2|||
+|Tumor_Validation_Allele1|||
+|Tumor_Validation_Allele2|||
+|Match_Norm_Validation_Allele1|||
+|Match_Norm_Validation_Allele2|||
+|Verification_Status|||
+|Validation_Status|||
+|Mutation_Status|||
+|Sequencing_Phase|||
+|Sequence_Source|||
+|Validation_Method|||
+|Score|||
+|BAM_File|||
+|Sequencer|||
+|t_ref_count|||
+|t_alt_count|||
+|n_ref_count|||
+|n_alt_count|||
+|HGVSc|||
+|HGVSp|||
+|HGVSp_Short|||
+|Transcript_ID|||
+|RefSeq|VEP||
+|Protein_position|VEP||
+|Codons|VEP||
+|Exon_Number|VEP||
+|gnomAD_AF|myvariant.info|Need to add "my_variant_info" in "enrichment_fields"|
+|gnomAD_AFR_AF|myvariant.info|Need to add "my_variant_info" in "enrichment_fields"|
+|gnomAD_AMR_AF|myvariant.info|Need to add "my_variant_info" in "enrichment_fields"|
+|gnomAD_ASJ_AF|myvariant.info|Need to add "my_variant_info" in "enrichment_fields"|
+|gnomAD_EAS_AF|myvariant.info|Need to add "my_variant_info" in "enrichment_fields"|
+|gnomAD_FIN_AF|myvariant.info|Need to add "my_variant_info" in "enrichment_fields"|
+|gnomAD_NFE_AF|myvariant.info|Need to add "my_variant_info" in "enrichment_fields"|
+|gnomAD_OTH_AF|myvariant.info|Need to add "my_variant_info" in "enrichment_fields"|
+|gnomAD_SAS_AF|myvariant.info|Need to add "my_variant_info" in "enrichment_fields"|
+|MA:FIS|Mutation Assessor|Need to add "mutation_assessor" in "enrichment_fields"|
+|MA:FImpact|Mutation Assessor|Need to add "mutation_assessor" in "enrichment_fields"|
+|MA:link.MSA|Mutation Assessor|Need to add "mutation_assessor" in "enrichment_fields"|
+|MA:link.PDB|Mutation Assessor|Need to add "mutation_assessor" in "enrichment_fields"|
+|Polyphen_Prediction|Polyphen|Need to add "polyphen" in "enrichment_fields|
+|Polyphen_Score|Polyphen|Need to add "polyphen" in "enrichment_fields|
+|SIFT_Prediction|SIFT|Need to add "sift" in "enrichment_fields|
+|SIFT_Score|SIFT|Need to add "sift" in "enrichment_fields|
+|Ref_Tri||Need to add "nucleotide_context" in "enrichment_field|
+|Var_Tri||Need to add "nucleotide_context" in "enrichment_field|
+|Annotation_Status|||
+
+### Enrichment fields
```

Comment:
Wondering if we should rephrase this header/section a bit. "Enrichment fields" is a genome nexus specific term, so maybe not super clear to a user? Another option is to say "Add additional annotation columns" or something along those lines and only mention that it involves setting `genomenexus.enrichment_fields=`

Some other points to consider here:

1. Good to indicate how to set this parameter on the command line for the compiled jar, rather than pre compilation phase. A user would prefer to use compiled source code rather than having to compile the entire project from source. Even better if there's a docker image, since they won't even need to install java. Many bioinformaticians would be familiar with how to use docker. See e.g. example here: https://docs.genomenexus.org/annotate-file
2. Since the term enrichment_fields is genome nexus specific it makes me wonder if it would be better for a user to eventually have a parameter like `--annotation-sources nucleotide_context,sift,mutation_assessor` etc. We could continue to use the term `enrichment_fields` in the code if we want, but for the user facing interfaces it might be better to use terminology that users are already familiar with

Reply:
btw: for 1. something like this might work:

```

java
    -Dgenomenexus.enrichment_fields=annotation_summary,my_variant_info
    -jar annotationPipeline/target/annotationPipeline-*.jar \
    -r \
    --filename test/data/minimal_example.in.txt  \
    --output-filename test/data/minimal_example.out.uniprot.txt \
    --isoform-override uniprot
```
It would be good if we can make that a proper command line parameter for the java app itself (like `--annotation-source xxx`) rather than a system env variable